packages/map/styles/Home.module.css

```
@@ -12,7 +12,7 @@
 }
 
 .title {
-  font-family: 'Saira';
+  font-family: 'Saira', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Saira" (font-family-name-quotes)

(font-family-name-quotes)